packages/alchemy/src/signer/session/manager.ts

```diff
@@ -0,0 +1,115 @@
+import { z } from "zod";
+import type { AlchemySignerClient } from "../client";
+import type { User } from "../client/types";
+import type { Session } from "./types";
+
+export const DEFAULT_SESSION_MS = 15 * 60 * 1000; // 15 minutes
+
+export const SessionManagerParamsSchema = z.object({
+  sessionKey: z.string().default("alchemy-signer-session"),
+  storage: z.enum(["localStorage", "sessionStorage"]).default("localStorage"),
+  expirationTimeMs: z
+    .number()
+    .default(DEFAULT_SESSION_MS)
+    .describe(
+      "The time in milliseconds that a session should last before expiring [default: 15 minutes]"
+    ),
+  client: z.custom<AlchemySignerClient>(),
+});
```

Comment:
thought we were going to remove zod? then should we keep adding new zod types?

Reply:
Ah yeah so I started writing these before the V3 refactor so this is now "legacy". In the interest of unblocking royal, I'll come back to this later. 

But yeah, in general let's not add more zod stuff because it gets hairy when generics are introduced.